### DISCUSSION TITLE
person reid with yolo model

### DISCUSSION THREAD

**Discussion by hanamuura** [ultralytics/ultralytics#16492] [Q&A] [answered]
i started to learn yolo docs for person reid and didnt find any models, which can do this. can anyone help me with this question please, and provide some example of usage or link how this can be used with yolo model (if it possible). ty all and sorry for waisting your time.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @hanamuura, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com) where you may find helpful resources for various applications, including example usage for models.

If your question pertains to integrating person re-identification with YOLO models, kindly share further details or specific requirements to assist our community and developers in providing the best guidance possible.

For an engaging community experience, you can join our [Discord](https://ultralytics.com/discord) for real-time chat, explore discussions on [Discourse](https://community.ultralytics.com), or engage with fellow enthusiasts on our [Subreddit](https://reddit.com/r/ultralytics) 🎧.

## Upgrade

Ensure you have the latest `ultralytics` package and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) installed in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

You can explore YOLOv8 in these verified environments:

- **Notebooks**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**: See our [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon**: Refer to the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: Follow the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing.

Please note that this is an automated response, but rest assured, an Ultralytics engineer will also assist you soon. 😊

**Comment by glenn-jocher** [comment]
YOLO models are primarily designed for object detection, not person re-identification. You might need to integrate YOLO with a separate re-identification model for this task. For more details, you can explore the YOLO documentation or community forums.

**Reply by hanamuura** [reply]
ty for ur answer. can u advice some free models for this task, if u know, please.
sorry for wasting ur time

**Reply by glenn-jocher** [reply, marked as answer]
@hanamuura i'm glad to help. For person re-identification, consider exploring specialized re-ID models available on platforms like PyTorch Hub or TensorFlow Model Garden.

**Reply by hanamuura** [reply]
ty for ur reply

**Reply by glenn-jocher** [reply]
@hanamuura you're welcome! If you have any more questions, feel free to ask.
